euler_support/urdf/euler_macro.xacro

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" ?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
+    <xacro:macro name="euler" params="prefix">
+        <xacro:include filename="$(find motoman_mh5lf_support)/urdf/mh5lf_macro.xacro" />
+        <xacro:motoman_mh5lf prefix="${prefix}motoman_" />
+        <!--xacro:include filename="$(find euler_support)/urdf/asus_xtion_pro.xacro" /-->
+        <xacro:include filename="$(find industrial_extrinsic_cal)/urdf/asus_sensor_cal_macro.xacro" />
+        <xacro:asus_sensor_definitions prefix="${prefix}asus_" parent_link="euler_motoman_base_link" />
 
-<xacro:include filename="$(find vetex_support)/urdf/vetex_macro.xacro" />
-<xacro:include filename="$(find motoman_mh5lf_support)/urdf/mh5lf_macro.xacro" />
-<!--xacro:include filename="$(find euler_support)/urdf/asus_xtion_pro.xacro" /-->
-<xacro:include filename="$(find industrial_extrinsic_cal)/urdf/asus_sensor_cal_macro.xacro" />
-<xacro:include filename="$(find euler_support)/urdf/eef_macro.xacro" />
+        <xacro:include filename="$(find euler_support)/urdf/eef_macro.xacro"/>
```

Comment:
Make sure that the indentation is using two space characters instead of tab characters.